docpipeline/generate.py

```diff
@@ -436,15 +408,13 @@ def build_blobs(blobs: List[storage.Blob]):
                         f"did you mean docfx-{blob.name}?"
                     )
                 )
-            process_blob(blob, devsite_template)
+            process_blob(blob, TEMPLATE_DIR)
```

Comment:
Just a thought - if it's available as a global constant and we don't expect this to change, could we simply use the global constant instead of passing it around?

Reply:
Done. Much better!